HIGHLIGHTS.md

```diff
@@ -4,6 +4,15 @@
 
 What highlights can you come up with to share with the class?
 
+- codecademy - provides a bit more code defintion, Jessica
+- w3schools.com - helps as a reference in overall Javascript, Alvin
+- css-tricks.com - helps with more css, a good walk through, Rob
+- Javascript for Dummies book - a good beginner's guide, Alejandro
+
+- codewithmosh.com - his free videos on Youtube are older, pay website is more recent
+- Youtube - in general, can be searched for videos on various subjects
+
+
```

Comment:
There are all amazing resources!!!! I will copy and pasta them into the `#ramdom` channel

Thank you!